lib/src/types/navigation_view_types.dart

```diff
@@ -439,3 +439,19 @@ class CameraChangedEvent {
   /// Current position of the camera.
   final CameraPosition position;
 }
+
+/// Represent map padding for the Google Maps view.
+/// {@category Navigation View}
+/// {@category Map View}
+class MapPadding {
```

Comment:
We shuold not need this if we use https://api.flutter.dev/flutter/painting/EdgeInsets-class.html